src/utils/link-handler.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+function link({ LINK_TITLES }) {
+  return function handler(h, node) {
+    const props = {};
+    if (node.url) {
```

Comment:
You are also rewriting external links, which does not seem like a good idea.

Reply:
Fixed in https://github.com/adobe/hypermedia-pipeline/pull/120/commits/6386202b2e004517b064f2ed2991b8dded8739e9